cylc/flow/cfgspec/suite.py

```diff
@@ -358,27 +358,37 @@
             supports use of the 365 (never a leap year) and 366 (always a leap
             year) calendars.
         ''')
-        Conf('runahead limit', VDR.V_STRING, desc='''
+        Conf('runahead limit', VDR.V_STRING, 'P5', desc='''
             Runahead limiting prevents the fastest tasks in a suite from
             getting too far ahead of the slowest ones, as documented in
             :ref:`RunaheadLimit`.
 
-            This config item specifies a hard limit as a cycle interval
-            between the slowest and fastest tasks. See also
-            :cylc:conf:`[..]max active cycle points` which defines the limit
-            as a number of cyles.
+            This limit on the number of consecutive spawned cycle points is
+            specified by an interval between the least and most recent: either
+            an integer (e.g. ``P3`` -  works for both :term:`integer cycling`
+            and :term:`datetime cycling`), or a time interval (e.g. ``PT12H`` -
+            only works for datetime cycling). Alternatively, if a raw number is
+            given, e.g. ``7``, it will be taken to mean ``PT7H``, though this
+            usage is deprecated.
 
-            Example: ``PT12H`` - for a 12 hour limit under ISO 8601 cycling.
-        ''')
-        Conf('max active cycle points', VDR.V_INTEGER, 3, desc='''
```

Comment:
Should the conf node item actually be left in for the sake of documentation (with a note that it's deprecated), or is removing it better?

Reply:
When we bring this back it will have to be with a different name so best to remove.